HomeWork1/inheritance/people/Person.java

```diff
@@ -44,4 +44,29 @@ public String getFullName(){
         return fullName;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Person person = (Person) o;
+
+        if (!name.equals(person.name)) return false;
+        if (!surname.equals(person.surname)) return false;
+        return patronymicName.equals(person.patronymicName);
+
+    }
+
+    @Override
+    public String toString() {
+        return String.format("{%2$s %1$s %3$s}", name, surname, patronymicName);
```

Comment:
use MessageFormat.format, it is more readable

Reply:
Ok.